File: Lab 1/Lab1 - Java/MyFirstAppJava/app/src/main/java/com/example/myfirstappjava/SecondActivity.java
package com.example.myfirstappjava;

import android.os.Bundle;
import android.widget.Button;
import android.widget.TextView;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class SecondActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_second);

        String message = getIntent().getStringExtra("EXTRA_MESSAGE");
        TextView sectextView = findViewById(R.id.textViewSecond);
        sectextView.setText(message);

        TextView textViewGreet = findViewById(R.id.secTextView);
        textViewGreet.setText("Welcome to the Second Activity!");

        Button buttonBack = findViewById(R.id.buttonBack);
        buttonBack.setOnClickListener(v -> finish());

    }
}

File: Lab 1/Lab1 - Kotlin/MyFirstAppKotlin/app/src/main/java/com/example/myfirstappkotlin/SecondActivity.kt
package com.example.myfirstappkotlin

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class SecondActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_second)

        val message = intent.getStringExtra("EXTRA_MESSAGE")
        val textView: TextView = findViewById(R.id.textViewSecond)
        textView.text = message

        val backButton: Button = findViewById(R.id.buttonBack)
        backButton.setOnClickListener{finish()}
    }
}